### ISSUE TITLE
Where can we use the interface for social scientists?

### ISSUE THREAD

**Issue by keyuchen21** [punyajoy/HateXplain#8]
In the Todos section, you guys mentioned there is an interface for social scientists, I'm wondering how can I access it?

![image](https://user-images.githubusercontent.com/54015474/139971276-0c71fa5c-e7a8-41fb-815f-3050d3f4cad3.png)

**Comment by punyajoy** [comment]
Yeah we are in the process of building huggingface spaces. Will notify once done
